pkg/pool-weighted/contracts/smart/InvestmentPool.sol

```diff
@@ -71,44 +82,76 @@ contract InvestmentPool is BaseWeightedPool, ReentrancyGuard {
         uint256[] endWeights
     );
     event SwapEnabledSet(bool swapEnabled);
+    event ManagementFeePercentageSet(uint256 managementFee);
```

Comment:
```suggestion
    event ManagementFeePercentageChanged(uint256 managementFee);
```

We have a `SwapFeePercentageChanged` event so it might make sense to use the same format here (although we do have a split between "set" and "changed" already)

Reply:
I made it "Set" because it's immutable, and "Changed" implies otherwise